routes/truckRoutes.js

```diff
@@ -1,2 +1,108 @@
-// Connect to mongodb in this module as this is where you'll be making creat/read/delete calls to your database
-// use 'mongodb://localhost/foodTruckAPI' for your mongoose connection string
\ No newline at end of file
+// Connect to mongodb in this module as this is where you'll be making
+// create/read/delete calls to your database
+// use 'mongodb://localhost/foodTruckAPI' for your mongoose connection string
+
+'use strict';
+// put your Express router code in here
+
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-undef */
+
+var express = require('express');
+var mongoose = require('mongoose');
+
+var Truck = require('../models/truckModel');
+
+var app = express();
+var db = mongoose.connect('mongodb://localhost/foodTruckAPI');
```

Comment:
While this works fine, better would be to put this in your server index as the db connection isn't exactly the concern of the router.